shard.yml

```
@@ -10,7 +10,7 @@ dependencies:
     version: ~> 0.4.0
   exception_page:
     github: crystal-loot/exception_page
-    version: ~> 0.3.0
+    version: ~> 0.4.0
```

I suppose we can go to the patch release directly?
```suggestion
    version: ~> 0.4.1
```

---

True, although in practice the end result will be the same, due to the `~>` operator logic.